content/en/docs/setup/install/multicluster/single-network/initial-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Initial Configuration
+description: Describes how to complete the initial configuration for a multicluster mesh on a flat network.
+weight: 10
+keywords: [multicluster,first-cluster]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, configure a
+single cluster {{< gloss >}}service mesh{{< /gloss >}} as your starting point.
+
+The single cluster mesh you deploy in this section, serves as a model for the
+configuration of the first {{< gloss >}}primary cluster{{< /gloss >}}. You need
+at least one primary cluster to complete the [different multicluster deployments.](/docs/ops/deployment/deployment-models/#multiple-clusters)
+
+Each cluster in a multicluster deployment needs a unique identifier. Using
+`CLUSTER_1` as a unique identifier for the first cluster, the following
+diagram shows a service mesh spanning a single primary cluster.
+
+{{< image width="75%"
+    link="init-conf.svg"
+    caption="A service mesh with a single primary cluster"
+    >}}
+
+Complete the [Before you begin instructions](/docs/setup/install/multicluster/#before-you-begin)
+before you continue.
+
+To set up a cluster as the primary cluster of a multicluster service
+mesh, complete the following configuration steps:
+
+1. [Configure trust for the cluster](#configure-trust)
+1. [Deploy Istio to the cluster](#deploy-istio)
+
+## Configure trust
+
+A multicluster service mesh deployment requires that you establish trust between
+the clusters in the mesh. To authenticate workloads across clusters, Istio
+requires that only one root Certificate Authority (CA) is active in the mesh. To
+establish shared trust, generate a single root CA for the entire mesh, and an
+intermediate CA for each cluster. Then, the shared root CA can sign the
+intermediate CA of each cluster.
+
+Production environments should use your organization's
+root CA whenever possible, or set up a secure PKI like [Vault PKI](https://www.vaultproject.io/docs/secrets/pki)
+to generate the common root certificate.
+
+The `Makefile` in [install/certs]({{< github_tree >}}install/certs)
+is a simple tool that helps generate the root certificate and intermediate
+certificates, but do not use it as a production-ready root CA. The tool
+lacks the following critical features:
+
+- Private key protection
+- Access control
+- Auditing
+- Monitoring
+
+Use the `Makefile` to generate the common root and the intermediate certificates
```

Comment:
This is outdated. We should not be using this Makefile anymore. Instead, we should be using the same `samples/certs` for every cluster. We're no longer generating intermediate certs, etc.